x-pack/plugins/security_solution/public/common/components/response_actions/endpoint_action_results.tsx

```diff
@@ -18,9 +19,20 @@ import { ResponseActionsEmptyPrompt } from './response_actions_empty_prompt';
 
 interface EndpointResponseActionResultsProps {
   action: LogsEndpointActionWithHosts;
+  isExpandableFlyout?: boolean;
 }
 
-export const EndpointResponseActionResults = ({ action }: EndpointResponseActionResultsProps) => {
+const StyledEuiComment = styled(EuiComment)`
```

Comment:
Is this styled component repeated in different places? If that's the case, could we create a separated component and reuse it?

Reply:
Thanks! Yes, we have exactly similar thing in 2 places. But one is used inside Osquery Plugin and other inside Security Solution. I didn't want to move it outside and then import from a different plugin in one place or another. 
What would you suggest?

Reply:
Ah, good point. Didn't noticed those were in different plugins. Then I guess it is ok having it duplicated as it is a small styled component. Other approach would be having ui extension points that can be added in one or other plugin. But I think it is not needed for this small use case.